pkg/apis/extensions/v1beta1/defaults.go

```diff
@@ -150,3 +151,7 @@ func SetDefaults_ReplicaSet(obj *ReplicaSet) {
 		*obj.Spec.Replicas = 1
 	}
 }
+
+func SetDefaults_NetworkPolicy(obj *NetworkPolicy) {
+	// CD4 TODO
```

Comment:
I'm OK to proceed with this as long as we get this in ASAP.  That said, we have very little default logic, it seems.